gello-world/src/components/login/login.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { FormErrors } from "./FormErrors.js";
+export class Login extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			username: "",
```

Comment:
When I learned of React it was in many examples as "basic" , because I decided that using this example would be very appropriate. Now  I`m using tutoriaol on  https://reactjs.org/ and doing same as.  I will find a way to facilitate readability but while I don`t understand what exactly needs to be done as a separate object.